domains/hajipur.json

```diff
@@ -0,0 +1,17 @@
+{
+    "repo": "https://github.com/Anushka-codergirl/MLHajipur",
+
+    "owner": {
+        "username": "Anushka-codergirl",
+        "email": "hajipur@mlcommunity.dev"
+    },
+
+    "target": {
+        "CNAME": {
+            "name": "hajipur",
+            "value": "Anushka-codergirl.github.io"
```

Comment:
Hey, the website at Anushka-codergirl.github.io seems to be a personal portfolio. We can provide subdomains only to community websites.